### PR TITLE
Add strings for AI TA landing page

### PR DESCRIPTION
**Pull request by kelbyhawn**
Adds strings for use on the new https://code.org/ai/teaching-assistant page. 

**Jira ticket:** [ACQ-1692](https://codedotorg.atlassian.net/browse/ACQ-1692)
**Figma mock:** [AI TA Landing Page V1](https://www.figma.com/file/5Pcxl4tcFgTeKDHZJUAQ5p/AI-TA-Launch?type=design&node-id=2373%3A2856&mode=design&t=ZBi6eBccRsHhqttE-1)

### CODE REVIEW

File: pegasus/sites.v3/hourofcode.com/i18n/en.yml [hunk 1]
```diff
@@ -10498,6 +10498,68 @@
...
+        answer: "The AI Teaching Assistant will launch to all CSD teachers in August 2024, in time for the 2024-2025 school year."
+      access:
+        question: "How do I access the AI Teaching Assistant?"
+        answer: "The AI Teaching Assistant is available for teachers accepted into the pilot in mini and end-of-unit projects in the Interactive Games and Animations unit. In those lessons, the AI Teaching Assistant will appear in the bottom left-hand corner of your screen."
+      keep_up:
+        question: "Where should I go to keep up with Code.org's investment in AI-supported tools?"
+        answer: "Code.org is investing in using AI to enhance teaching and learning.  Check out [https://code.org/ai](https://code.org/ai) to learn more about our AI curriculum, professional learning, and tools."
+      demo:
+        question: "Can I see a demo of how AI TA works?"
+        answer: "Join our curriculum manager, Angelina as she introduces the AI Teaching Assistant."
```
Comment (TurnerRiley) [inline pegasus/sites.v3/hourofcode.com/i18n/en.yml:10559]: Nit: comma after "Angelina"

File: pegasus/sites.v3/hourofcode.com/i18n/en.yml [hunk 2]
```diff
@@ -10498,6 +10498,68 @@
...
+        answer: "The AI Teaching Assistant is launching in CSD's Interactive Games and Animations Unit. The team is hard at work building AI-supported features for additional curriculum."
+      available:
+        question: "When will the AI Teaching Assistant be available to all Interactive Games and Animation users?"
+        answer: "The AI Teaching Assistant will launch to all CSD teachers in August 2024, in time for the 2024-2025 school year."
+      access:
+        question: "How do I access the AI Teaching Assistant?"
+        answer: "The AI Teaching Assistant is available for teachers accepted into the pilot in mini and end-of-unit projects in the Interactive Games and Animations unit. In those lessons, the AI Teaching Assistant will appear in the bottom left-hand corner of your screen."
+      keep_up:
+        question: "Where should I go to keep up with Code.org's investment in AI-supported tools?"
+        answer: "Code.org is investing in using AI to enhance teaching and learning.  Check out [https://code.org/ai](https://code.org/ai) to learn more about our AI curriculum, professional learning, and tools."
```
Comment (TurnerRiley) [inline pegasus/sites.v3/hourofcode.com/i18n/en.yml:10556]: Nit: should this be "[code.org/ai](https://code.org/ai)"? I feel like elsewhere on the site we omit the `https://` in the text shown

**Review by TurnerRiley — APPROVED** [review]
Looks great! Just two tiny nits

File: pegasus/sites.v3/hourofcode.com/i18n/en.yml (hunk 2 above)
Comment (kelbyhawn) [reply pegasus/sites.v3/hourofcode.com/i18n/en.yml:10556]: Good catch!

**Pull request merged**